index.html

```diff
@@ -14,41 +15,139 @@
+          <li><i class="fa-solid fa-m"></i></li>
+          <li><i class="fa-brands fa-square-github"></i></li>
+          <li><i class="fa-regular fa-hand-peace"></i></li>
+        </ul>
+      </section>
     </section>
 
-    <article class="job">
-      <p>
-        I’m a software developer! I can help you build a product , feature or
-        website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hesitate to contact me.
-      </p>
-    </article>
-    <section class="contact">
-      <p>LET’S CONNECT</p>
+    <section class="cards">
+
+      <div class="card">
+
+        <img class="cardimg" src="./images/tonic.png" alt="">
+
+        <div class="grid-item-title-1">
+          <p class="title">Tonic</p>
+          <div class="canopy">
+            <p>CANOPY</p>
+            <ul>
+              <li> Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <article>A daily selection of privately personalized reads;
+            no accounts or sign-ups required.</article>
+        </div>
+
+        <section class="button">
+          <ul>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>Javascript</li>  
+          </ul>
+          <div class="proyect">
+            <a href="#"><button>See Proyect</button></a>
+          </div>
+        </section>
+
+      </div>
+      <div class="multi"></div>
+
+      
+
+      <div class="tonic2"></div>
+      <div class="multi2"></div>
     </section>
-    <section class="socialmedia">
-      <ul>
-        <li><i class="fa-brands fa-twitter"></i></li>
-        <li><i class="fa-brands fa-linkedin"></i></li>
-        <li><i class="fa-solid fa-m"></i></li>
-        <li><i class="fa-brands fa-square-github"></i></li>
-        <li><i class="fa-regular fa-hand-peace"></i></li>
-      </ul>
+    <section class="aboutmyself">
+      <h2>About Myself</h2>
+
+      <article class="job">
+        <p>
+          Hello I’m a software developer! I can 
+          help you build a product , feature or 
+          website. Look through some of my 
+          work and experience! If you like what 
+          you see and have a project you need 
+          coded, don’t hesitate to contact me.
+        </p>
+      </article>
+      <section class="contact">
+        <p>LET’S CONNECT</p>
+      </section>
+      <div class="get">
+        <a href="#"><button>Get my resume</button></a>
+      </div>
+
+      <section class="socialmedia">
+        <ul>
+          <li><i class="fa-brands fa-twitter"></i></li>
+          <li><i class="fa-brands fa-linkedin"></i></li>
+          <li><i class="fa-solid fa-m"></i></li>
+          <li><i class="fa-brands fa-square-github"></i></li>
+          <li><i class="fa-regular fa-hand-peace"></i></li>
+        </ul>
+      </section>
+
+      <section class="lenguage">
+        <h2>
+          Lenguage
+        </h2>
+      
```

Comment:
- [ ] Instead of writing `lengauge` it grammatically corrects to write `langagues`. Kindly correct that 💯